html-forms_zoo/index.html

```diff
@@ -20,21 +20,53 @@ <h2 id="form-heading">Zoo Visitor Sign Up</h2>
         <fieldset>
           <legend>Enter your information</legend>
 
-          <!-- Email: label & input -->
+          <label for="input-email">enter your E-mail-Adress</label>
+          <input type="email" id="input-email" name="email" />
 
-          <!-- Date: label & input -->
+          <label for="input-date">When do you want to visit us?</label>
+          <input type="date" id="input-date" name="date" />
 
           <fieldset class="radio-options">
             <legend>Which one of these animals would you like to visit?</legend>
 
-            <!-- Radio Buttons: labels & inputs -->
+            <label for="sloths">Relaxed Sloths</label>
+            <input type="radio" id="sloths" name="attraction" value="sloths" />
+            <label for="kangaroos">Jumpy Kangaroos</label>
+            <input
+              type="radio"
+              id="kangaroos"
+              name="attraction"
```

Comment:
the name attribute should be different for each checkbox. If not, when using JS you will not know which one was selected

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox